packages/ui/react-ui-geo/src/components/Map/Map.tsx

```diff
@@ -117,7 +145,15 @@ const MapCanvas = forwardRef<MapController, MapCanvasProps>(({ markers, center,
   }, [markers]);
 
   return (
-    <div ref={ref} className='flex w-full h-full overflow-hidden bg-baseSurface'>
+    <div ref={ref} role='none' className='grid inset-0 overflow-hidden bg-baseSurface'>
+      {/* Focus ring. */}
+      <div
+        className={mx(
+          'z-[9999] absolute inset-0 border border-transparent pointer-events-none',
+          hasFocus && 'border-primary-500 ',
```

Comment:
Focus has a semantic tokens, `--dx-neutralFocusIndicator` and `--dx-accentFocusIndicator`, by default elements apply the neutral one. Avoid defining something new and special here if possible.